assets/about.css

```diff
@@ -0,0 +1,440 @@
+}
+
+.about-header {
+  font-size: 3rem;
+  font-weight: 800;
+  color: #ec240d;
+}
+
+.header-para {
+  background-color: #fff;
+}
+
+.about-email {
+  font-weight: bold;
+  text-decoration: underline;
+  margin-top: 3rem;
+  justify-content: center;
+}
+
+.about-section {
+  text-align: center;
+  justify-content: center;
+  padding: 3rem;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.1);
+}
+
+.logo-heading {
+  font-size: 2rem;
+}
+
+.about-para {
+  margin: 2rem 25rem;
+  font-size: 1.1rem;
+  color: var(--color-primary);
+}
+
+hr {
+  margin: 1rem auto;
+  width: 30px;
+  border: 1px solid #ec240d;
+}
+
+.footer-section {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 30vh;
+  gap: 2rem;
+  text-align: left;
+  background: rgb(51, 50, 50);
+  color: white;
+}
+
+.copyright-img {
+  width: 100px;
+  height: 150px;
+}
+
+.footer-para {
+  font-family: var(--alternate-font);
+  font-weight: 900;
+  font-size: 1.5rem;
+  text-align: center;
+}
+
+.footer-descr {
+  align-items: center;
+  justify-content: center;
+  color: var(--color-white);
+}
+
+.footer-section span {
+  color: var(--color-white);
+}
+
+/* Mobile-Version */
+@media screen and (max-width: 768px) {
+  .page-heading {
+    display: none;
+  }
+
+  .navbar {
+    display: flex;
+    justify-content: flex-start;
+    background: transparent;
+    position: absolute;
+    top: 0;
+    margin-top: 0;
+    box-shadow: none;
+  }
+
+  .logo-img {
+    display: none;
+  }
+
+  .nav-links {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    top: 0;
+    background: #ec5342ec;
+    height: 100vh;
+    width: 200px;
+    left: -200px;
+    text-align: left;
+    z-index: 2;
+    transition: 1s;
+    -webkit-backdrop-filter: blur(5px);
+    backdrop-filter: blur(5px);
+  }
+
+  .nav-items {
+    flex-direction: column;
+    padding-top: 5rem;
+    line-height: 5rem;
+    letter-spacing: 1.5px;
+  }
+
+  .nav-items li a {
+    font-size: 2rem;
+    color: rgb(225, 225, 225);
+  }
+
+  .nav-items li a:hover {
+    color: #ebc934;
+  }
+
+  .btn {
+    display: flex;
+    justify-content: center;
+    color: rgb(225, 225, 225);
+    border: 2px solid rgb(225, 225, 225);
+    padding: 1rem 2rem;
+    font-size: 1.4rem;
+    margin-top: 1.5rem;
+  }
+
+  i#openBtn,
+  i#closeBtn {
+    display: block;
+  }
+
+  .fa-times {
+    display: block;
+    color: var(--color-primary);
+    position: absolute;
+    left: 1rem;
+    font-size: 2rem;
+    margin: 1.2rem;
+  }
+
+  .fa-bars {
+    display: block;
+    color: var(--color-primary);
+    font-size: 3rem;
+    position: absolute;
+    left: 2.3rem;
+  }
+
+  /* Headline-section */
+  .hero-section {
+    height: 120vh;
+    width: 100%;
+    background-repeat: no-repeat;
+    object-fit: contain;
+    background-size: 100%;
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    margin: 10px;
+    background-color: rgb(250, 250, 250);
+  }
```

Comment:
- [ ] I suggest you fix this overlapping issue here aswell.

`about.css`
![image](https://user-images.githubusercontent.com/52143815/175438856-b1446fc4-a428-48ea-bc90-bb2d092917b5.png)